modules/social_features/social_follow_content/config/install/flag.flag.follow_post.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
```

Comment:
Do we need a new flag here? I realise that `follow_content` may not be entirely accurate anymore in name. However, they are functionally equivalent so I don't think that separating them makes much sense? Is there any functional requirement to distinguish the two?

Reply:
@Kingdutch Yes, we should add a new flag because one flag can be related only to one entity type.